doc/how-to/image/f_bench_autotile_labels.png

```diff
@@ -0,0 +1 @@
+/home/int/contribute/cdda/CDDA-Tilesets/templates/f_bench_autotile_labels.png
```

Comment:
This doesn't seem like something you intended to push

Reply:
It was forgotten in https://github.com/I-am-Erk/CDDA-Tilesets/commit/3e00b3ee45aca383346481bddad9eccca6b40da0 and I fixed it :) It's not loaded currently at https://i-am-erk.github.io/CDDA-Tilesets/how-to/autotiles.html , look just above the `Things like benches` title.

Reply:
it wasn't indeed. Anyway, fixed now.